### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by Siedlerchr**

### CODE REVIEW

File: src/main/resources/l10n/JabRef_zh_CN.properties [hunk 1]
```diff
@@ -2515,7 +2525,20 @@ This\ could\ indicate\ that\ JabRef\ did\ not\ shut\ down\ cleanly\ last\ time\
 
 Use\ the\ field\ FJournal\ to\ store\ the\ full\ journal\ name\ for\ (un)abbreviations\ in\ the\ entry=使用FJournal字段记录Journal全称
 
+Library\ to\ import\ into=Library to import into
 
+Multiline=Multiline
 
+Unable\ to\ open\ linked\ eprint.\ Please\ set\ the\ eprinttype\ field=Unable to open linked eprint. Please set the eprinttype field
```
Comment (calixtus) [inline src/main/resources/l10n/JabRef_zh_CN.properties:2532]: This does not seem to be a chinese translation...

File: src/main/resources/l10n/JabRef_zh_CN.properties [hunk 2]
```diff
@@ -1250,6 +1255,7 @@ Please\ specify\ a\ password=请指定密码
 Proxy\ configuration=代理服务器设置
 Use\ custom\ proxy\ configuration=使用自定义 HTTP 代理
 Proxy\ requires\ authentication=代理服务器需要密码
+Password\ kept\ for\ this\ session\ only.=Password kept for this session only.
```
Comment (calixtus) [inline src/main/resources/l10n/JabRef_zh_CN.properties:1258]: Not a translation

File: src/main/resources/l10n/JabRef_zh_CN.properties [hunk 3]
```diff
@@ -40,6 +42,8 @@ Abbreviations=缩写
 Unabbreviate\ journal\ names\ of\ the\ selected\ entries=展开选中记录的缩写期刊名称
 Unabbreviated\ %0\ journal\ names.=展开 %0 期刊名称。
 
+dotless=dotless
+shortest\ unique=shortest unique
```
Comment (calixtus) [inline src/main/resources/l10n/JabRef_zh_CN.properties:46]: not a translation

File: src/main/resources/l10n/JabRef_zh_CN.properties (hunk 1 above)
Comment (Siedlerchr) [reply src/main/resources/l10n/JabRef_zh_CN.properties:2532]: Comment that on crowdin at the string

**Comment by koppor** [comment]
We will keep this PR open until the issues are fixed (and we have a contributor wanting that his changes appear under his name. Thus, we need to merge that manually).

File: src/main/resources/l10n/JabRef_zh_CN.properties (hunk 1 above)
Comment (koppor) [reply src/main/resources/l10n/JabRef_zh_CN.properties:2532]: Removed translation on Crowdin.

File: src/main/resources/l10n/JabRef_zh_CN.properties (hunk 2 above)
Comment (koppor) [reply src/main/resources/l10n/JabRef_zh_CN.properties:1258]: Removed translation on Crowdin.

File: src/main/resources/l10n/JabRef_zh_CN.properties (hunk 3 above)
Comment (koppor) [reply src/main/resources/l10n/JabRef_zh_CN.properties:46]: Removed translation on Crowdin.

File: src/main/resources/l10n/JabRef_zh_CN.properties [hunk 4]
```diff
@@ -2515,7 +2525,20 @@ This\ could\ indicate\ that\ JabRef\ did\ not\ shut\ down\ cleanly\ last\ time\
...
 Use\ the\ field\ FJournal\ to\ store\ the\ full\ journal\ name\ for\ (un)abbreviations\ in\ the\ entry=使用FJournal字段记录Journal全称
 
+Library\ to\ import\ into=Library to import into
 
+Multiline=Multiline
 
+Unable\ to\ open\ linked\ eprint.\ Please\ set\ the\ eprinttype\ field=Unable to open linked eprint. Please set the eprinttype field
+Unable\ to\ open\ linked\ eprint.\ Please\ verify\ that\ the\ eprint\ field\ has\ a\ valid\ '%0'\ id=Unable to open linked eprint. Please verify that the eprint field has a valid '%0' id
+Main\ directory\ not\ found=Main directory not found
+Please\ select\ a\ valid\ main\ directory\ under=Please select a valid main directory under
```
Comment (koppor) [inline src/main/resources/l10n/JabRef_zh_CN.properties:2535]: This is not a valid sentence, is it? What is the "under"? DEnglish maybe? 😅

File: src/main/resources/l10n/JabRef_zh_CN.properties (hunk 1 above)
Comment (Siedlerchr) [reply src/main/resources/l10n/JabRef_zh_CN.properties:2532]: Maybe also add a hint that Jabref will auto use English if the string is not translated

**Comment by Siedlerchr** [comment]
Regarding the main file directory under, I think it's missing a Parameter
or it's referring to the path in the preferences.
Definitely needs to be adjusted as other languages have different word
orders

Oliver Kopp ***@***.***> schrieb am Sa., 29. Apr. 2023, 00:52:

**Comment by koppor** [comment]
> Regarding the main file directory under,

PR is at https://github.com/JabRef/jabref/pull/9814

**Pull request merged**